### DISCUSSION TITLE
What is the correct way to implement organization-level permissions for multi-tennant applications?

### DISCUSSION THREAD

**Discussion by aantn** [supabase/supabase#2313] [Questions]
Hello everyone,
I am evaluating Supabase for a multitennant SaaS application. Every table has an organization_id column and every user is associated with exactly one organization via a UserToOrganization table. I would like to implement row-level permissions such that users can only update/insert rows belonging to their organization.

Is it possible to implement this with Supabase? Can I embed a generic query in the authentication policies that joins between the user and UserToOrganization table? Alternatively, is it possible to directly store a user's organization on the user itself?

What is the recommended way to implement this?

**Comment by tristandeschler** [comment]
@aantn - were you ever able to figure this piece out? Working on a similar concept and can’t find any content.

**Comment by aantn** [comment]
No, as far as I understand it's not supported yet.
